.github/workflows/pull_request.yml

```diff
@@ -47,6 +47,29 @@ jobs:
       - name: Install yarn deps
         if: steps.cache-yarn.outputs.cache-hit != 'true'
         run: yarn
+  check-schemas:
```

Comment:
I'm not sure on the name check schemas, at first I thought it might be the validator schema check. Maybe adding the word `version` helps.

Reply:
Changed, thanks!